api/egl.xml

```diff
@@ -1055,6 +1055,10 @@
     <enums namespace="EGL" start="0x34D0" end="0x34DF" vendor="ANGLE" comment="Reserved for Peng Huang - ANGLE (via github pull request)">
             <unused start="0x34D0" end="0x34DF"/>
     </enums>
+
```

Comment:
<enums namespace="EGL" start="0x34F0" end="0x34FF" vendor="ANGLE" comment="Reserved for Ken Russell - ANGLE (via github pull request)">
            <unused start="0x34F0" end="0x34FF"/>
    </enums>